regolith/templates/former.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% block header %}
+<h1>Former Members</h1>
+{% endblock header %}
+{% block content %}
+{% for p in people if p.get('active', False) %}
```

Comment:
Shouldn't this be `if not p.get('active', True)`?  That former members are those whose active setting is False, and the default active setting is True?